backend/src/main/java/ai/verta/modeldb/authservice/PublicRoleServiceUtils.java

```diff
@@ -266,6 +268,18 @@ public WorkspaceDTO getWorkspaceDTOByWorkspaceId(
     return Collections.emptyList();
   }
 
+  @Override
+  public boolean createWorkspaceRoleBinding(
+          Long workspaceId,
+          WorkspaceType workspaceType,
```

Comment:
Should we add a method with workspace name? I can see this one being useful for migration, since these are the two fields we have in the db, but during the API call, we'll have just the name. We can let UAC resolve by itself what is the corresponding ID if we send the name.

Reply:
It's an easy overload to add